From 8b11d81cfdc486c22e6aee35f4dca60469ce7102 Mon Sep 17 00:00:00 2001
From: Leonard Crestez <leonard.crestez@nxp.com>
Date: Mon, 18 Jun 2018 21:37:40 +0300
Subject: [PATCH 3989/5242] MLK-18628-2 ARM: dts: imx7d-sdb: Restore &pwm1
 node

commit  97ef560492eb85cd736d92d13764e4ec5f9e4aaf from
https://source.codeaurora.org/external/imx/linux-imx.git

This was removed by upstream commit 5eaeaccdaedb
("ARM: dts: imx7d-sdb: Pass 'enable-gpios' and 'power-supply' properties")

That commit claims that GPIO1_IO01 is not a PWM because it's connected
to a pin labelled "PWREN" of J14 connector. However that pin does behave
as a pwm despite the name.

The exact same pattern happens with imx6ul-evk which has an LCD8000-43T
display on J901.

Signed-off-by: Leonard Crestez <leonard.crestez@nxp.com>
Reviewed-by: Fabio Estevam <fabio.estevam@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d-sdb.dts |    6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm/boot/dts/imx7d-sdb.dts b/arch/arm/boot/dts/imx7d-sdb.dts
index 1c6e2b2..d1a655a 100644
--- a/arch/arm/boot/dts/imx7d-sdb.dts
+++ b/arch/arm/boot/dts/imx7d-sdb.dts
@@ -625,6 +625,12 @@
 	status = "okay";
 };
 
+&pwm1 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_pwm1>;
+	status = "okay";
+};
+
 &pcie {
 	pinctrl-names = "default";
 	reset-gpio = <&extended_io 1 GPIO_ACTIVE_LOW>;
-- 
1.7.9.5

